solutions/swb-ui/src/api/datasets.ts

```diff
@@ -3,11 +3,11 @@
  *  SPDX-License-Identifier: Apache-2.0
  */
 
+import { httpApiGet } from '@aws/workbench-core-swb-common-ui';
 import useSWR from 'swr';
 import { DatasetItem } from '../models/Dataset';
-import { httpApiGet } from './apiHelper';
 
-const useDatasets = () => {
+const useDatasets = (): { datasets: DatasetItem[], areDatasetsLoading: boolean } => {
   const { data, isValidating } = useSWR(() => 'datasets', httpApiGet);
```

Comment:
`useSWR` has typings we can use. It would look something like `useSWR<dataType, errorType>()`. This would prevent the data object from being `any` type

Reply:
Skipping for now, since the ultimate return type of the method is defined.